deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -0,0 +1,273 @@
+Transform: 'AWS::Serverless-2016-10-31'
+Description: >
+  - Creates a series of lambdas and a step function to process manifests.
+  - The lambdas create iiif manifest and pyramid tiffs out of images.
+  - The finished product are copied to a bucket to serve manifests out of and an image server
+    to display the iiif images.
+  - Creates a bucket for to serve manifests out of
+  - Creates a bucket that the step functions use as an in process location.
+  - Requires the iiif-service and mellon-data-broker pipelines to be built.  Uses ssm keys from those services
+Parameters:
+  EnvType:
+    Type: String
+    Default: 'test'
+  ImageSourceBucket:
+    Type: 'AWS::SSM::Parameter::Value<String>'
+    Description: The name of the source bucket that the IIIF Image service will read from for its assets.
+    Default: '/all/stacks/mellon-data-broker/publicbucket'
+  ImageServerUrl:
+    Type: 'AWS::SSM::Parameter::Value<String>'
+    Description: The url of the IIIF Image Server
+    Default: '/all/stacks/mellon-image-service/url'
+  InfrastructureStackName:
+    Type: String
+    Default: "mellon-app-infrastructure"
+    Description: The name of the parent infrastructure/networking stack that you created. Necessary
+                 to locate and reference resources created by that stack.
+
+Outputs:
+  StateMachine:
+    Value: !Ref StateMachine
+
+  ProcessBucket:
+    Value: !Ref ProcessBucket
+
+  ManifestBucket:
+    Value: !Ref ManifestBucket
+
+Resources:
+  ProcessBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      LoggingConfiguration:
+        DestinationBucketName:
+          Fn::ImportValue: !Join [':', [!Ref InfrastructureStackName, 'LogBucket']]
+        LogFilePrefix: s3/data-broker/
+
+  ManifestBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: PublicRead
+      LoggingConfiguration:
+        DestinationBucketName:
+          Fn::ImportValue: !Join [':', [!Ref InfrastructureStackName, 'LogBucket']]
+        LogFilePrefix: s3/data-broker/
+      WebsiteConfiguration:
```

Comment:
Is anything directly getting things from the bucket via http? I would have expected your client to have issues because this isn't served via https. This should probably be hosted via CloudFront, which means removing this web config.

Reply:
yes it is a future enhancement to get this put this behind a cloud front.